test/models/location_test.rb

```diff
@@ -61,6 +61,17 @@ class LocationTest < ActiveSupport::TestCase
     end
   end
 
+  test "top_countries does not include least common countries" do
+    stub_geocoder("Tokoyo", country: "China")
```

Comment:
Let's stick with a location that exists in China, like you did for Mumbai in India. It  helps reduce the mental hurdle of wondering if you meant "Tokyo" or Japan. :D